bindings.c

```diff
@@ -3794,6 +3799,90 @@ static uint64_t get_reaper_age(pid_t pid)
 	return procage;
 }
 
+/*
+ * Returns 0 on success.
+ * It is the caller's responsibility to free `return_usage`, unless this
+ * function returns an error.
+ */
+static int read_cpuacct_usage_all(char *cg, char *cpuset, struct cpuacct_usage **return_usage)
+{
+
+	int cpucount = get_nprocs();
+	struct cpuacct_usage *cpu_usage;
+	int rv = 0, i, j, ret, read_pos = 0, read_cnt;
```

Comment:
Nit: If you do `int rv = -1` here you can spare all those `rv = -1` in your error paths and simply return and also remove `{`, `}`. :)

Reply:
That's true, but I prefer to make the return value explicit, e.g. someone can change the default `rv` in the future and it won't break.